apt_sysvinit-utils.dockerfile

```diff
@@ -3,14 +3,25 @@ ARG OS_TYPE
 
 FROM $OS_TYPE:$BASE_IMAGE_TAG
 
+# Re-declare OS_TYPE & BASE_IMAGE_TAG ARGS
+ARG OS_TYPE
+ARG BASE_IMAGE_TAG
+
 ENV container docker
 ENV DEBIAN_FRONTEND noninteractive
+ENV OS_TYPE=$OS_TYPE
+ENV BASE_IMAGE_TAG=$BASE_IMAGE_TAG
```

Comment:
no need to set as ENV variable, you can simply access the args as you have done below.

Reply:
Can you pls refer: https://github.com/puppetlabs/litmusimage/pull/95#discussion_r1822713158

Reply:
It seems to be working https://github.com/puppetlabs/litmusimage/actions/runs/11598903565/job/32295848548?pr=95#step:10:593
Thanks.